draft-ietf-taps-impl.md

```diff
@@ -184,6 +184,26 @@ Any one of these sub-entries on the aggregate connection attempt would satisfy t
 
 The step of gathering candidates involves identifying which paths, protocols, and endpoints may be used for a given Connection. This list is determined by the requirements, prohibitions, and preferences of the application as specified in the Selection Properties.
 
+### Gathering Endpoint Candidates
+
+Both Local and Remote Endpoint Candidates must be discovered during connection establishment.  To support ICE, or similar protocols, that involve an out-of-band indirect signalling exchange to exchange candidates with the Remote Endpoint, it’s important to be able to query the set of candidate Local Endpoints, and give the protocol stack a set of candidate Remote Endpoints, before it attempts to establish connections.
```

Comment:
Nit: `... exchange to exchange candidates` repeats the word `exchange` probably more than we ought to